### ISSUE TITLE
Support for specifying Tag?

### ISSUE THREAD

**Issue by xdansmith** [neutmute/nlog-targets-loggly#9]
Great library!

The Loggly library allows you to specify tags which are great for filtering by Source inside loggly.

```
            var logEvent = new LogglyEvent();
            logEvent.Options.Tags.Add("android");
```

Is there any way to specify a tag using this library, or could this be added? I can submit a PR for the functionality if it isn't planned but there is interest for it.

**Comment by neutmute** [comment]
Yes - this target library uses https://github.com/neutmute/loggly-csharp you can specify tags at a global level via config or statically.
But you possibly mean to customise tags at a specific log event level? If so I don't know off the top of my head how to do that but if a PR is required, it would be welcome!

**Comment by neutmute** [comment]
There is this ability to add data to an event 
https://github.com/neutmute/nlog-targets-loggly/blob/master/src/NLog.Targets.Loggly/LogglyTarget.cs#L76

It doesn't translate to tags (I don't think - i havent used this lib for a while) but I was using it to filter for a 'theme' of logs that were interwoven among others.
That might be of use?

If not, a PR would be fine but please submit a couple of dot points on your proposed changes/architecture
cheers!

**Comment by zpavic** [comment]
I'm just in the process of evaluating Loggly and this library for use in our application and I think that we'd have a need to specify tags for some loggers.

How about something like this:
```
<nlog>
  <extensions>
    <add assembly="Nlog.Targets.Loggly" />
  </extensions>
  <targets>
    <target name="Loggly" xsi:type="Loggly" layout="${defaultLayout} />
    <target name="TaggedLoggly" xsi:type="Loggly" layout="${defaultLayout} tags="audit" />
  </targets>
  <rules>
    <logger name="AuditLogger" minlevel="Info" writeTo="TaggedLoggly" final="true" />
    <logger name="*" minlevel="Info" writeTo="Loggly" />
  </rules>
</nlog>
```
The TaggedLoggly layout would append the tags defined on the target:
```
logglyEvent.Data.Add("message", logMessage);
foreach (var tag in Tags)
{
    logglyEvent.Options.Tags.Add(tag);
}

foreach (var key in logEvent.Properties.Keys)
{
    logglyEvent.Data.AddIfAbsent(key.ToString(), logEvent.Properties[key]);
}
           
loggly.Log(logglyEvent);
```

@xdansmith I'd think we'd be interested in this if you are still game for a PR.

**Comment by neutmute** [comment]
looks sensible to me!

**Comment by snakefoot** [comment]
Created PR #16

**Comment by snakefoot** [comment]
@neutmute Resolved with #16
